src/components/TradeWidget/TokenRow.tsx

```diff
@@ -268,6 +270,7 @@ const TokenRow: React.FC<Props> = ({
       </div>
       <InputBox>
         <InputWithTooltip
+          autoFocus={!readOnly && autoFocus}
```

Comment:
I did it as this first, but then I thought it would be more declarative with an optional field "focus". 

It's a bit strange at a purist level to "focus" in every "TokenRow" that's editable. We can do this because we have knowledge that there's ONLY two rows.

However, it doesn't look there's going to be more than two in the future.